src/deps/zcl_abaplint_deps_find.clas.abap

```diff
@@ -326,12 +543,14 @@ CLASS ZCL_ABAPLINT_DEPS_FIND IMPLEMENTATION.
             CASE lv_clstype.
               WHEN '0'.
                 APPEND INITIAL LINE TO ct_tadir ASSIGNING <ls_tadir>.
-                <ls_tadir>-ref_obj_type = 'CLAS'.
+                <ls_tadir>-ref_obj_type = 'CLASS'.
```

Comment:
sure? obj_type should be char4

Reply:
yes it should be. As the field is only 4 long, it does not make a difference.  Will correct it.